skins/cat17/address-form/src/store.ts

```diff
@@ -2,7 +2,7 @@ import Vue from 'vue';
 import Vuex, {StoreOptions} from 'vuex';
 import {Validity} from './lib/validation_states';
 import {Helper} from './mixins/helper';
-import { AddressState, InputField, Payload, PostData, ValidationResult} from './types';
+import { AddressState, AddressValidity, InputField, Payload, PostData, ValidationResult} from './types';
```

Comment:
Does TS complain if you don't import this here? It's already in `AddressForm.vue` :thinking: